docs/java-application-development/how-to-use-dummy-signature-service.md

```diff
@@ -0,0 +1,15 @@
+# How to use the Dummy Container Signature Validation Service
+
+The Dummy Container Signature Validation Service is an example implementation of the [Container Signature Validation Service interface] which mainly serves as a reference for future implementations and testing.
```

Comment:
Is it a missing link?

Reply:
Addressed in 4481e238efb3fb9143d8e6b2b44ee3588f5bd5bc